### ISSUE TITLE
Base Flow

### ISSUE THREAD

**Issue by erikperin** [Vialerta/Core#2]
Here the base stream.
It is open to suggestions...

[base_flow.html.zip](https://github.com/Vialerta/core/files/480704/base_flow.html.zip)

**Comment by erikperin** [comment]
Is there any way to check the compatibility of sensors and cameras on mobile devices via javascript?

**Comment by soumfilipe** [comment]
> Is there any way to check the compatibility of sensors and cameras on mobile devices via javascript?

What do you mean for compatibility? Check if there is a camera or gyro sensor, for example?

**Comment by erikperin** [comment]
Yes @cfilipes ... and they permissions...

To check:
- GPS
- Gyro
- Camera (Photo and Video)
- Mic

And make it compatible with TS and Voice Recognition. Like... Google Now (mobile and desktop versions).

**Comment by mblodorn** [comment]
In android you can ask for these permissions when installing the app.

**Comment by erikperin** [comment]
But... and the Web Client?

**Comment by mblodorn** [comment]
Why would we need a mobile web client if we gonna build an app? Web cliente mobile could be for consultation only. :)

**Comment by erikperin** [comment]
No... all the clients run with all features... (expect the legacy version)

**Comment by EltonCarreiro** [comment]
Actually whe have an API called Media Devices which is provided by some browsers like Firefox, not sure about other browsers. Almost sure they dont have support but if it works in FF and Chrome I  think it would be enough.

**Comment by EltonCarreiro** [comment]
Looks like FF and Chrome works :)

https://developer.mozilla.org/pt-BR/docs/Web/API/Navigator/mediaDevices

**Comment by erikperin** [comment]
Great... 😃
